actors/abi/sector.go

```diff
@@ -200,6 +197,7 @@ type PoStCandidate struct {
 }
 
 type PoStProof struct { //<curve, system> {
+	RegisteredProof
```

Comment:
Do you anticipate a similar thing being required for SealProof? 

That will conflict with #174 (but we should land this to signal/avoid that regression). Also invalidates #187. cc @magik6k

Reply:
As far as I saw, seal was not broken suring the refactor performed when you moved the actors, so it will should be fine.

Reply:
@anorth 

> Do you anticipate a similar thing being required for SealProof?

I do not. It looks like `SealVerifyInfo.OnChain.RegisteredProof` represents the `SealProof`'s `RegisteredProof`, so we should be good to go.

Reply:
@porcuquine 

> As far as I saw, seal was not broken during the refactor performed when you moved the actors, so it should be fine.

@porcuquine roger that, thanks for doing that archaeology for me